src/clientversion.cpp

```diff
@@ -116,17 +116,46 @@ std::string FormatSubVersion(const std::string &name, int nClientVersion, const
     if (!subverOverride.empty())
         return subverOverride;
 
+    // sanitize comments per BIP014, format user agent and check total size
+    std::vector<std::string> uacomments = {};
+    for (std::string &cmt : mapMultiArgs["-uacomment"])
+    {
+        if (cmt != SanitizeString(cmt, SAFE_CHARS_UA_COMMENT))
+            break;
+        uacomments.push_back(SanitizeString(cmt, SAFE_CHARS_UA_COMMENT));
+    }
+    // If this is a 32bit build then append an identifier since we'd like to know
+    // how many still run this configuration.
+    {
+        int temp = 0;
+        int *ptemp = &temp;
+        if (sizeof(ptemp) == 4)
+            uacomments.push_back("32bit");
```

Comment:
fair points. Since I see this as a temporary thing to asses the proportion of 32bit nodes among all nodes, I would rather adding it in front of `uacomments` vector along with a code comments that explain the rationale.